h/streamer/websocket.py

```diff
@@ -71,7 +71,7 @@ def handle_message(message):
             jsonschema.validate(payload, filter.SCHEMA)
 
             # Add backend expands for clauses
-            _expand_clauses(payload)
+            _expand_clauses(socket.request, payload)
```

Comment:
I think it should be the same due to the way that `scoped_session` works, but I think we should probably check.